playbooks/vars/maas-managed-k8.yml

```diff
@@ -0,0 +1,19 @@
+---
+# Copyright 2017, Rackspace US, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+maas_managed_k8_kube_config_dir: /root/.k8
+
+maas_managed_ko_kubectl_release: v1.9.1
```

Comment:
`s/ko/k8` ?

Does this mean anytime we update `kubectl` we have to update this value?

Reply:
Done